### ISSUE TITLE
Default margin on figure

### ISSUE THREAD

**Issue by prtksxna** [samwilson/embed-wikimedia#6]
So apparently the default browser CSS adds margin to `figure`s? 
<img width="1134" alt="Screenshot 2019-07-26 at 9 14 02 AM" src="https://user-images.githubusercontent.com/9491/61924529-d6c11500-af85-11e9-8d19-05d7dd2ff161.png">

Do we want to do something about this?

**Comment by samwilson** [comment]
Yes that does look a bit odd. Is it the same in the front end? I'm happy to add custom styling for this. Maybe figures aren't meant to be included verbatim in the editor? Probably it's not how the image block does things.

**Comment by prtksxna** [comment]
I don't see it happening for normal images though, do you?
